src/components/Sing/ExportOverlay.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div v-if="exportingInfo.nowExporting" class="export-overlay">
+    <div>
+      <QSpinner color="primary" size="2.5rem" />
+      <div class="q-mt-xs">
+        {{
+          nowRendering
+            ? "レンダリング中・・・"
+            : `${exportingInfo.mediaName}を書き出し中・・・`
+        }}
+      </div>
+      <!-- NOTE: 書き出しのキャンセルはレンダリング中にのみ可能 -->
+      <QBtn
+        v-if="nowRendering"
+        padding="xs md"
+        :label="`${exportingInfo.mediaName}の書き出しをキャンセル`"
+        class="q-mt-sm"
+        outline
+        @click="cancelExport"
+      />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { useStore } from "@/store";
+import { ExhaustiveError } from "@/type/utility";
+
+type ExportingInfo =
+  | { nowExporting: false }
+  | { nowExporting: true; mediaName: "音声" }
+  | { nowExporting: true; mediaName: "labファイル" };
+
+const store = useStore();
+
+const nowRendering = computed(() => {
+  return store.state.nowRendering;
+});
+
+const exportingInfo = computed((): ExportingInfo => {
```

Comment:
（全く問題ないのですが）
`ref<T>`と書くことが多いので、合わせておくとちょっと読みやすいかも！
まあ本当に見やすさだけなので勝手に変えさせていただきます！
```suggestion
const exportingInfo = computed<ExportingInfo>(() => {
```